### ISSUE TITLE
Bug fixes refactored website

### ISSUE THREAD

**Issue by pr130** [CorrelAid/hugo-website#142]
- [x] bootstrap buttons on mobile look weird
- [x] on project pages, the "CorrelAid Team" frame is cut off at the left
- [x] add action buttons under each paragraph on "volunteer" page 
- [x]  testimonial view on mobile is not a good experience (see #135 ) (wontfix for now)
- [x] "book appointment" sounds weird -> "get a 1:1 meeting" + "arrange meeting" on button 
- [x] link to podcast on english page as well
- [x] link to podcast only once on german page
- [x] testimionials from volunteers on volunteer page and not on about-us / work
- [x] font size at prize and awards extremely huge on mobile
- [x] events menu entry does not appear on mobile
- [x] The mobile version does not has a   recommendation about signing up for the correlaid newsletter.
- [x] several links are broken in blog posts (see @Heba-azzam 's word doc) 
- [x] links on landing page are broken (header of "features") 
- [x] new buttons on volunteer page look werid.
